packages/sdk/src/context.ts

```diff
@@ -9,9 +9,7 @@ export interface Contextual<Self extends Contextual<Self>> {
 
 export interface Context extends Value.Struct {
   visitor_id?: string;
-  openFeature?: Value.Struct & {
-    targeting_key?: string;
-  };
+  targeting_key?: string;
```

Comment:
Do we even want semantic context for targeting_key? Do we ever expect that to be a thing in Confidence?

Reply:
For later...